crates/rpc/types/src/types.rs

```diff
@@ -126,25 +127,46 @@ impl Default for RpcL1Status {
     }
 }
 
+/// In reference to checkpointed client state tracked by the CSM.
 #[derive(Clone, Debug, Deserialize, Serialize)]
 pub struct RpcClientStatus {
     /// Blockchain tip.
+    // TODO remove this since the CSM doesn't track this anymore, we're pulling it in indirectly
     #[serde(with = "hex::serde")]
+    #[deprecated(note = "no longer tracked by client state")]
     pub chain_tip: [u8; 32],
 
     /// L1 chain tip slot.
+    // TODO remove this since the CSM doesn't track this anymore, we're pulling it in indirectly
+    #[deprecated(note = "no longer tracked by client state")]
```

Comment:
We should remove all these deprecated RPCs for testnet 1.

Reply:
Yes definitely.  I'll create tickets for removing these.  I didn't want to expand the scope further by going and actually removing all the uses so I just added the deprecated attr to make noise.

Reply:
Created as STR-1087.